### PR TITLE
Updated the Habitat 2.0 citation

### PR DESCRIPTION
**Pull request by ASzot**
## Motivation and Context

The recently released Habitat 2.0 paper needs to be in the references and citations list.

## How Has This Been Tested

Visually inspected rendered README.

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by aclegg3 — APPROVED** [review]

**Review by mathfac — APPROVED** [review]
🚀🚀🚀

**Comment by dhruvbatra** [comment]
@ASzot -- let's also add the bibtex to the "Citing Habitat" section.

**Comment by ASzot** [comment]
Now added.

**Comment by Skylion007** [comment]
Same comment as Hab-Lab. I personallly would prefer the newer citation first. Thoughts?

**Review by Skylion007 — CHANGES_REQUESTED** [review]
Same comment as before.

File: README.md [hunk 1]
```diff
@@ -58,6 +58,12 @@ If you use the Habitat platform in your research, please cite the following [pap
   booktitle =     {Proceedings of the IEEE/CVF International Conference on Computer Vision (ICCV)},
   year      =     {2019}
 }
+@article{szot2021habitat,
```
Comment (Skylion007) [inline README.md:61]: Consider moving this bibtex to the top.

**Comment by aclegg3** [comment]
CI doesn't like this PR. 🤷

**Pull request closed without merging**